map2loop/sampler.py

```diff
@@ -139,12 +139,14 @@ def sample(
                 targets = []
 
             # For the main cases Polygon and LineString the list 'targets' has one element
-            for target in targets:
+            for i, target in enumerate(targets):
                 df2 = pandas.DataFrame(columns=schema.keys()).astype(schema)
                 distances = numpy.arange(0, target.length, self.spacing)[:-1]
                 points = [target.interpolate(distance) for distance in distances]
                 df2["X"] = [point.x for point in points]
                 df2["Y"] = [point.y for point in points]
+                df2['segNum'] = i
```

Comment:
should segNum actually be 'segementID'

Reply:
We were quick to decide and that is what we settled on (and is already in loopprojectfile).  If we were changing it to be more verbose I'd make it segmentId as I prefer camelcase :p

Reply:
Happy to change; @RoyThomsonMonash did you want me to change this in LPF too?

Reply:
If you want to change it in LPF you can, just need to delete the 0.1.0 versions on anaconda when the 0.1.1 version with the name change is up